designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -48,6 +48,7 @@ package object definition {
       // FE need this information because branch parameters aren't changed dynamically during node validation so they never
       // should be invalidated
       branchParam: Boolean,
+      requiredParam: Boolean,
```

Comment:
It isn't used by the fronted, is it? We should avoid adding to API parameters that are not used by the FE. From a developer of this project perspective, this would be just a dead code. Dead code == to be removed. 
Solutions that I have in mind:
- Can you base on validators.exists(mandatory) rule instead on your side?
- Another API with described responsibility

Reply:
I added comment that this param is used by external project